durabletask/task.py

```diff
@@ -71,6 +71,20 @@ def is_replaying(self) -> bool:
         """
         pass
 
+    @property
+    @abstractmethod
+    def custom_status(self) -> str:
+        """Get the custom status.
+        """
+        pass
+
+    @custom_status.setter
+    @abstractmethod
+    def custom_status(self, custom_status: str) -> None:
```

Comment:
Can we do `set_custom_status` instead of a `custom_status` property setter? This is how our other SDKs work (using a method rather than a property), and gives us greater flexibility in the future if we want to add parameters or make behavior changes that wouldn't be natural with a property.

Reply:
done